src/server/server_family.cc

```diff
@@ -2845,7 +2851,7 @@ void ServerFamily::ReplicaOfInternal(CmdArgList args, Transaction* tx, SinkReply
   }
```

Comment:
It will be more clear I think if you enter the Drakarys call under if (on_err == ActionOnConnectionFail::kReturnOnError)instead of if (tx)

Reply:
I did not want to change the semantics. I will update on this

Reply:
oh Drakarys needs, nvm